core/server/data/schema/schema.js

```diff
@@ -72,7 +72,9 @@ module.exports = {
         meta_title: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 300}}},
         meta_description: {type: 'string', maxlength: 2000, nullable: true, validations: {isLength: {max: 500}}},
         email_subject: {type: 'string', maxlength: 300, nullable: true},
-        frontmatter: {type: 'text', maxlength: 65535, nullable: true}
+        frontmatter: {type: 'text', maxlength: 65535, nullable: true},
+        feature_image_alt: {type: 'text', maxlength: 65535, nullable: true},
```

Comment:
I'm not sure if giving full `65535` characters is necessary for an "alt" text. The HTML spec doesn't put a limit on it, but the general rule that floats around is having around 125 characters to describe an image and using longdesc attribute for detailed descriptions. Do you have any references for this>

Reply:
This is the best I could find on the "125" number reason - https://moz.com/learn/seo/alt-text

Reply:
Good point. 125 sounds reasonable and looks widely referenced. The only other reference to a specific length I could find is a [WCAG 2.0 test suite for alt text](https://www.w3.org/WAI/GL/WCAG20/tests/test3.html) which will fail if text is >100chars but allows for longer if there's a good reason. In the [WAI images tutorial](https://www.w3.org/WAI/tutorials/images/complex/) they have an `alt` text with 143 chars but it looks like an outlier with alternative ways to reference a longer description.

Initially I went with `text` and the larger fields to avoid problems with row length. Do we have any issues around that with the `posts_meta` table?

Reply:
Changed to a `191` length string column with max length validation of `125` chars